### ISSUE TITLE
Now that TimeDistributedMerge is removed, what is the replacement of it?

### ISSUE THREAD

**Issue by ymcui** [fchollet/keras#2271]
I have experienced the latest version of Keras in github, and noticed that there is a big difference from previous versions.
`TimeDistributed` has now been separated as a `wrapper`.
`TimeDistributedDense` can be replaced by `TimeDistributed(Dense())`
how about the `TimeDistributedMerge`?
should we write an `Lambda` layer by ourselves?

**Comment by carlthome** [comment]
Doesn't `TimeDistributed(Merge())` work?

**Comment by ymcui** [comment]
@carlthome 
in the older version, you can put an 3D tensor in `TimeDistributedMerge()`.
you suggest to use `TimeDistributed(Merge())`, but i don't think `Merge` layer will receive only one argument of 3D tensor.

**Comment by ymcui** [comment]
@carlthome 
i just wondering if `Merge` layer can take no argument in 1.0.0.
for older version, `Merge` can not be used in format of `Merge()` with nothing in brackets.

**Comment by around1991** [comment]
@carlthome: the Merge layer merges several tensors into one, whereas TimeDistributedMerge just collapsed one axis of a single tensor.

**Comment by fchollet** [comment]
The replacement is to use the functional API to build your models:
http://keras.io/getting-started/functional-api-guide/

**Comment by around1991** [comment]
Sorry, I don't get how one would do sum-pooling (say) on the output of an LSTM with return_sequences=True in the new API?

**Comment by zhipengChen** [comment]
the functional API cannot build TimedistributedMerge

**Comment by DingKe** [comment]
@around1991 I think TimeDistributedMerge is removed because it can be implemented via Lambda layer, like:
```python
time_distributed_merge_layer = Lambda(function=lambda x: K.mean(x, axis=1), 
                   output_shape=lambda shape: (shape[0],) + shape[2:])
```

**Comment by pxlong** [comment]
@DingKe  Hi

```
lstm_2 = LSTM(128, return_sequences=True)(dropout_1)
lstm_goal = LSTM(128, return_sequences=True)(masking_goal)
merge_direction = TimeDistributed(Merge([lstm_2, lstm_goal], mode='concat'))
```

If I want to merge two tensors (like codes showed above), how to write a lambda layer? Since the lambda layer take one argument, how to concatenate two tensors?
